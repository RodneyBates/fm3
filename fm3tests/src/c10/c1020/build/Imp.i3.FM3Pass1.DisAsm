 ItkEOF()
 ItkRightEnd()
 ItkInterfaceLtPatch(C4,L3(16_3),[1,0])
 ItkInterfaceRt(L3(16_3),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkBlockDeclListLtPatch(C10,L2(16_2),[3,2])
 ItkBlockDeclListRt(L2(16_2),[3,2])
 ItkBlockDeclListSepPatch(C43,L1(16_1),[4,0])
 ItkConstDeclLtPatch(C43,[4,8])
 ItkConstDeclColonPatch(C49,[4,11],FALSE)
 ItkConstDeclEqualsPatch(C51,[4,11])
 ItkConstDeclRt([4,8])
 ItkIntLit(16_4(4),[4,13])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id3"I2",[4,8])
 ItkConstDeclLtPatch(C10,[3,8])
 ItkConstDeclColonPatch(C16,[3,11],FALSE)
 ItkConstDeclEqualsPatch(C18,[3,11])
 ItkConstDeclRt([3,8])
 ItkIntLit(16_17(23),[3,13])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id2"I1",[3,8])
 ItkLookupScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
