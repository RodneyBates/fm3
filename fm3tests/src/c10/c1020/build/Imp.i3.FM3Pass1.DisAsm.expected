 ItkEOF()
 ItkRightEnd()
 ItkInterfaceLtPatch(C4,L3(16_3),[1,0])
 ItkInterfaceRt(L3(16_3),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkOpenDeclListLtPatch(C10,L2(16_2),[3,2])
 ItkOpenDeclListRt(L2(16_2),[3,2])
 ItkConstDeclLtPatch(C58,[4,8])
 ItkConstDeclRt([4,8])
 ItkLiteral(96 StkIntLit,(16_4),[4,13])
 ItkConstDeclValue([4,11])
 ItkDeclTypeAbsent([4,11])
 ItkConstDeclType([4,11])
 ItkDeclId(L9(16_9),Id3"I2",L-9223372036854775808(16_8000000000000000),[4,8])
 ItkOpenDeclListSep(L1(16_1),[3,2])
 ItkConstDeclLtPatch(C10,[3,8])
 ItkConstDeclRt([3,8])
 ItkLiteral(96 StkIntLit,(16_17),[3,13])
 ItkConstDeclValue([3,11])
 ItkDeclTypeAbsent([3,11])
 ItkConstDeclType([3,11])
 ItkDeclId(L9(16_9),Id2"I1",L-9223372036854775808(16_8000000000000000),[3,8])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
